Modix.Services/GuildStats/GuildStatService.cs

```diff
@@ -79,13 +80,13 @@ public Task Handle(UserLeft notification, CancellationToken cancellationToken)
         }
 
         /// <inheritdoc />
-        public async Task<IReadOnlyDictionary<GuildUserEntity, int>> GetTopMessageCounts(IGuild guild)
+        public async Task<IReadOnlyCollection<PerUserMessageCount>> GetTopMessageCounts(IGuild guild, ulong userId)
         {
             var key = GetKeyForMsgCounts(guild);
```

Comment:
You need to add the user id to the cache key now if the stats are tailored to them, otherwise it'll display the same thing for each member in the guild.

Reply:
Ah, you're absolutely right. Thanks for catching that. Fixed.